chevah/brink
# [#4444] Add coverator for publishing coverage reports.

Scope
=====

Update brink to use coverator when publishing coverage reports.


Changes
=======

I've updated the coverage_publish method.

I've also added a configuration to SETUP to set the address where to upload reports.

Also fixed the revision property in BrinkGit which was returning the tree, not the revision hash.


How to try and test the changes
===============================

reviewers: @adiroiban 

Please see if changes make sense.
